cmd/crc/cmd/version.go

```diff
@@ -1,3 +1,19 @@
+/*
+Copyright (C) 2019 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
```

Comment:
we haven't added this to any of the files.
Perhaps we should AFTER/WHEN we also run a `go fmt` against the repo